WcaOnRails/app/controllers/api/v0/persons_controller.rb

```diff
@@ -1,4 +1,4 @@
-# frozen_string_literal: true
+  # frozen_string_literal: true
```

Comment:
Why are you adding a space here? Seems like some local editor config that is doing shenanigans on auto-format. Also makes me wonder why our Ruby code linter doesn't catch this, it's usually very very whiny about stuff like this.